docs/topics/testing-rules.adoc

```diff
@@ -158,7 +158,10 @@ The `<iterable-filter>` element has no unique child elements.
 [id="classification_exists_syntax_{context}"]
 ==== <classification-exists> syntax
 
-The `<classification-exists>` element determines if a specific classification title has been included in the analysis. For additional information, see the link:{LinkAPI}org/jboss/windup/reporting/config/ClassificationExists.html[ClassificationExists] class.
+The `<classification-exists>` element determines if a specific classification title has been included in the analysis. For additional information, see the link:{LinkAPI}org/jboss/windup/reporting/config/ClassificationExists.java[ClassificationExists] class.
+
+ink:{LinkAPI}org/jboss/windup/rules/general/IterableFilter.java[IterableFilter] class.
```

Comment:
You have clearly failed to check the preview

![image](https://github.com/windup/windup-documentation/assets/106804821/27368292-0eab-482c-b6c2-d008b45f5166)

Reply:
Removed.